packages/cozy-harvest-lib/package.json

```
@@ -73,6 +73,7 @@
   "peerDependencies": {
     "cozy-client": "^14.4.0",
     "cozy-device-helper": "^1.10.2",
+    "cozy-flags": "^2.3.5",
```

```suggestion
    "cozy-flags": "^2.4.0",
```

tant qu'à faire pour mettre des warnings et obliger les app à upgrader ? :)

---

Ba justement, j'avais vu la version différente, et j'avais laissé 2.3.5 parce qu'harvest n'a pas besoin forcément de cozy-flags 2.3.5. A mon avis il faut quand même essayer de ne déclencher des warnings qui ne sont pas forcément pertinent, sinon on risque de ne plus les regarder ces warnings.